packages/orm/src/EntityManager.ts

```diff
@@ -181,6 +190,21 @@ export interface FlushOptions {
  */
 export type EntityManagerMode = "read-only" | "in-memory-writes" | "writes";
 
+export type FindOperation =
+  | typeof findOperation
+  | "findPaginated"
+  | typeof findByUniqueOperation
+  | typeof findCountOperation
+  | typeof lensOperation
+  | typeof loadOperation
+  | typeof manyToManyLoadOperation
+  | typeof manyToManyFindOperation
+  | typeof oneToManyLoadOperation
+  | typeof oneToManyFindOperation
+  | typeof oneToOneLoadOperation
+  | typeof populateOperation
+  | typeof recursiveChildrenOperation
+  | typeof recursiveParentsOperation;
```

Comment:
I'm a little worried this is going to end up being too-granular, but 🤷 shouldn't hurt

Reply:
I think the only one we actually would care about right now is `findCount`